src/kOS.Safe/Screen/ScreenBuffer.cs

```diff
@@ -38,6 +44,12 @@ public class ScreenBuffer : IScreenBuffer
         public int RowCount { get; private set; }
         public int ColumnCount { get; private set; }
 
+        /// <summary>Delegate prototype expected by AddResizeNotifier</summary>
+        /// <param name="sb">This screenbuffer telling the callback who it is</param>
+        /// <returns>telling this screenbuffer how many vertical rows to scroll as a result of the resize.</returns>
+        public delegate int ResizeNotifier(IScreenBuffer sb);
+        protected List<ResizeNotifier> notifyees;
```

Comment:
The notifyees are delegates that need to be told whenever this ScreenBuffer gets its size changed.  This is mostly so that the SubBuffers and the TermWindow attached to this ScreenBuffer get told about it and recalculate themselves.  There is no direct reference from SubBuffer to ScreenBuffer, so the SubBuffer normaly can't query its ScreenBuffer to find out its size.  So instead the ScreenBuffer registers the SubBuffer's notifier method with itself when it adds a SubBuffer.